pkg/graphdb/graphdb.go

```diff
@@ -378,12 +378,22 @@ func (db *Database) Purge(id string) (int, error) {
 		tx.Rollback()
 		return -1, err
 	}
-
 	changes, err := rows.RowsAffected()
 	if err != nil {
 		return -1, err
 	}
 
+	// Clear who's using this id as parent
+	refs, err := tx.Exec("DELETE FROM edge WHERE parent_id = ?;", id)
```

Comment:
Can't SQLite handle this by itself if a proper foreign-key constraint is defined?